rosdep/python.yaml

```
@@ -6551,6 +6552,11 @@ python3-gnupg:
   ubuntu: [python3-gnupg]
 python3-google-cloud-storage-pip: *migrate_eol_2025_04_30_python3_google_cloud_storage_pip
 python3-google-cloud-texttospeech-pip: *migrate_eol_2025_04_30_python3_google_cloud_texttospeech_pip
+python3-googleapi:
+  debian: [python3-googleapi]
+  ubuntu:
+    bionic: [python3-googleapi]
+    focal: [python3-googleapi]
```

Sorry, I missed this on the last review.  There is no need to split this up, so just make this:

```suggestion
  ubuntu: [python3-googleapi]
```

---

Thank you for your kind review, fixed